dbdialog/src/main/java/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -272,6 +276,12 @@ public void databaseTypeRemoved( String pluginName ) {
   private XulTextbox iamProfileName;
   protected XulMenuList namedClusterList;
 
+  //Azure SQL DB Variables
+  private XulMenuList azureSqlJdbcAuthMethod;
+  private XulCheckbox azureAlwaysEncryptionEnabled;
```

Comment:
call out azure SQL DB specifically everywhere and not a generic azure names as in the future we might have more azure integrations